.github/workflows/chromatic.yml

```diff
@@ -31,6 +31,15 @@ jobs:
       # Root yarn installs all workspaces (root, common, dotcom)
       - uses: bahmutov/npm-install@v1
 
+      - name: Chromatic - Common Rendering
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN__COMMON_RENDERING }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          workingDir: common-rendering
```

Comment:
using `workingDir` means we're able to remove the output dir flag. Chromatic automatically creates and then finds the `storybook-static` folder within the project.